app/views/pages/dashboard.html.erb

```diff
@@ -2,11 +2,11 @@
   What's available in Find
 <% end %>
 
-<h1 class="heading-large">
-  What's available in Find right now
-</h1>
-
 <main role="main" id="content">
+  <h1 class="heading-large">
+    What's available in Find right now
```

Comment:
this doesn't look like an apostrophe.

Reply:
There is indifferent usage across all of our content. I didn't want to address that here when the focus is on heading sizes.

Incidentally referring to "Find" here is inconsistent with the reset of the site where we say "data.gov.uk" or "Find open data".

Reply:
Fair enough, I'd missed the fact it was already there.